setup.py

```diff
@@ -91,7 +91,6 @@ def get_package_data(package):
     packages=get_packages(package),
     package_data=get_package_data(package),
     cmdclass={'test': PyTest},
-    install_requires=install_requires,
```

Comment:
It looks like there are valid requirements, is there a reason they're not shown here?